torch/ao/sparsity/sparsifier.py

```diff
@@ -0,0 +1,167 @@
+
+import abc
+import copy
+
+import torch
+from torch import nn
+from torch.nn.utils import parametrize
+
+from .parametrization import MulBy
+
+SUPPORTED_MODULES = {
+    nn.Linear
+}
+
+def _module_to_path(model, layer, prefix=''):
+    for name, child in model.named_children():
+        new_name = prefix + '.' + name
+        if child is layer:
+            return new_name
+        child_path = _module_to_path(child, layer, prefix=new_name)
+        if child_path is not None:
+            return child_path
+    return None
+
+def _path_to_module(model, path):
```

Comment:
Do both these functions work for sequentials? Also, for consistency suggest calling this _fqn_to_module